### DISCUSSION TITLE
reset/rebuild

### DISCUSSION THREAD

**Discussion by everythingness** [flowershow/flowershow#606] [General and Help]
somewhere along the line a discrepancy has crept in to the notes flowershow/obsidian thinks are on my website and what is actually there. is this possible? it seems to be because i deleted a note via flowershow and it didn't actually get deleted. so i was wondering if there is a simple way to reset or rebuild everything from scratch or should i just delete the repository and make a new one?

**Comment by rufuspollock** [comment]
@everythingness you can go to the git repo that your obsidian is syncing with and remove the file directly via the github interface. That should fix the problem.
